src/test/ui/async-await/issues/issue-65419/issue-65419-async-fn-resume-after-panic.rs

```diff
@@ -6,7 +6,6 @@
 // error-pattern: thread 'main' panicked at '`async fn` resumed after panicking'
 // edition:2018
 // ignore-wasm no panic or subprocess support
-// ignore-emscripten no panic or subprocess support
```

Comment:
Please keep this for the missing subprocess support.

Reply:
To be honest, I don't understand the test completely, but it looks to me that it is not using processes at all. I think the comment is only there for copy & paste reasons.

Reply:
Right, my bad.